public/public.js

```diff
@@ -102,20 +213,23 @@ jQuery(document).ready(function () {
     });
     validator.form()
 
-    // This is a form delay to discourage robots
-    let counter = 7;
-    let myInterval = setInterval(function () {
-        let button = jQuery('#submit-button')
+    let button = jQuery('#submit-button')
+    button.html( window.TRANSLATION.submit ).prop('disabled', false)
 
-        button.html( window.TRANSLATION.submit_in + ' ' + counter)
-        --counter;
+    // // This is a form delay to discourage robots
+    // let counter = 7;
+    // let myInterval = setInterval(function () {
+    //     let button = jQuery('#submit-button')
 
-        if ( counter === 0 ) {
-            clearInterval(myInterval);
-            button.html( window.TRANSLATION.submit ).prop('disabled', false)
-        }
+    //     button.html( window.TRANSLATION.submit_in + ' ' + counter)
+    //     --counter;
+
+    //     if ( counter === 0 ) {
+    //         clearInterval(myInterval);
+    //         button.html( window.TRANSLATION.submit ).prop('disabled', false)
+    //     }
 
-    }, 1000);
+    // }, 1000);
```

Comment:
comments on this section?

Reply:
This should have been uncommented. I turned it off during testing, must have forgot to turn it back on.